patches/minecraft/net/minecraft/client/gui/screens/controls/KeyBindsList.java.patch

```diff
@@ -1,5 +1,27 @@
 --- a/net/minecraft/client/gui/screens/controls/KeyBindsList.java
 +++ b/net/minecraft/client/gui/screens/controls/KeyBindsList.java
+@@ -29,6 +_,8 @@
+    final KeyBindsScreen f_193858_;
+    int f_193859_;
+ 
++   final int NAME_SPLIT_LENGTH = 185;
++
+    public KeyBindsList(KeyBindsScreen p_193861_, Minecraft p_193862_) {
+       super(p_193862_, p_193861_.f_96543_ + 45, p_193861_.f_96544_, 20, p_193861_.f_96544_ - 32, 20);
+       this.f_193858_ = p_193861_;
+@@ -43,10 +_,11 @@
+             this.m_7085_(new KeyBindsList.CategoryEntry(Component.m_237115_(s1)));
+          }
+ 
++         // NeoForge: max width for the keybind descriptions to make all readable
```

Comment:
This comment should be the line above the `this.f_193859_ = Math.min(i, NAME_SPLIT_LENGTH);` so it is more clear what we are changing. Also the format for the comment should be `// Neo: max width...`

Reply:
Perfect, I was just trying to put the comment above the altered sections altered and it was originally a much larger edit.